code/src/types/common.ts

```diff
@@ -22,3 +22,11 @@ export type APIError = {
   title: string;
   detail: string;
 };
+
+export type RTKQueryReturnState<T = null> = {
+  data: T | null;
+  isLoading: boolean;
+  isSuccess: boolean;
+  isError: boolean;
+  error: null;
+};
```

Comment:
I suggest also adding optional generic for error

Reply:
Thanks for suggestion. I added error generic, it can be useful later